tests/ModuleTest.php

```diff
@@ -61,15 +55,16 @@ public function shouldRunCommandOriginalPredisClientsCommandSuccessfully() : voi
 
     /**
      * @test
-     * @expectedException \Redislabs\Exceptions\InvalidCommandException
      */
-    public function shouldFailForMagicMethodCall() : void
+    public function shouldFailForMagicMethodCall(): void
     {
         $redisClient = new Predis\Client();
         /**
          * @var Module
          */
         $module = Module::createWithPredis($redisClient);
-        $module->invalidMethod('test');
+        $this->assertThrows(InvalidCommandException::class, function () use ($module) {
```

Comment:
can we just use this assert instead of assertThrows:

`$this->expectException(InvalidCommandException::class);`